server/staff/urls.py

```diff
@@ -37,4 +37,6 @@
     url(r'^school/account/info/$', views.SchoolAccountInfoView.as_view(), name="school_account_info"),
     url(r'^school/price/config/$', views.SchoolPriceConfigView.as_view(), name="school_price_cfg"),
     url(r'^school/account/income/records/$', views.SchoolIncomeRecordView.as_view(), name="school_income_records"),
+    url(r'^region/(?P<rid>\d+)/$', views.RegionView.as_view(), name="region_show"),
+    url(r'^region/(?P<rid>\d+)/(?P<action>open)/$', views.RegionView.as_view(), name="region_open"),
```

Comment:
It's better to move the action `open` into the query parameter like `https://.../region/(?P<rid>\d+)/?action=open`, so only one url rule is sufficient. After all, url should describe a resource, not actions.